person.rb

```diff
@@ -1,23 +1,28 @@
-require_relative 'nameable'
+require_relative './nameable'
```

Comment:
- [ ] The file in your project is named `namable` not `nameable,` it's throwing an error when I run the app. Kindly fix this issue 👍